docs(security): init (#1612)
ReVanced/revanced-manager: SECURITY.md

<p align="center">
  <picture>
    <source
      width="256px"
      media="(prefers-color-scheme: dark)"
      srcset="assets/revanced-headline/revanced-headline-vertical-dark.svg"
    >
    <img 
      width="256px"
      src="assets/revanced-headline/revanced-headline-vertical-light.svg"
    >
  </picture>
  <br>
  <a href="https://revanced.app/">
       <picture>
           <source height="24px" media="(prefers-color-scheme: dark)" srcset="assets/revanced-logo/revanced-logo-round.svg" />
           <img height="24px" src="assets/revanced-logo/revanced-logo-round.svg" />

>        </picture>

Reviewer: These files are not present in the `compose-dev` branch.

Author: Done:tm:

         <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032213-faf25ab8-0bc3-4a94-a730-b524c96df124.png" />
      </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://x.com/revancedapp">
      <picture>
         <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/93124920/270180600-7c1b38bf-889b-4d68-bd5e-b9d86f91421a.png">
         <img height="24px" src="https://user-images.githubusercontent.com/93124920/270108715-d80743fa-b330-4809-b1e6-79fbdc60d09c.png" />
      </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://www.youtube.com/@ReVanced">
      <picture>
         <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
         <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
     </picture>
   </a>
   <br>
   <br>
   Continuing the legacy of Vanced
</p>

# 🔒 Security Policy

> This document describes how to report vulnerability to ReVanced Manager.

Reviewer: ```suggestion
This document describes how to report security vulnerabilities for ReVanced Manager.
```

## ⏳ Supported Versions

| Version | Branch      | Supported          |
| ------- | ------------|------------------- |
| v1.18.0 | main        | :white_check_mark: |
| latest  | dev         | :white_check_mark: |

> | latest  | compose-dev | :white_check_mark: |

Reviewer: Maybe this should be below the "Reporting a Vulnerability" section

## 🚨 Reporting a Vulnerability

Please open an issue in our [advisory tracker](https://github.com/ReVanced/revanced-manager/security/advisories/new) or reach out privately to us on [Discord](https://discord.gg/revanced).

> If a vulnerability is confirmed and accepted, you can join us [Discord](https://discord.gg/revanced) server to receive an special contributor role.

Reviewer: ```suggestion
If a vulnerability is confirmed and accepted, you can our [Discord](https://discord.gg/revanced) server to receive a special contributor role.
```

Author: > ```suggestion
> If a vulnerability is confirmed and accepted, you can our [Discord](https://discord.gg/revanced) server to receive a special contributor role.
> ```

Do you mean "you can **join** our..."
